spec/mailers/previews/user_preview.rb

```diff
@@ -0,0 +1,6 @@
+# Preview all emails at http://localhost:3000/rails/mailers/user
+class UserPreview < ActionMailer::Preview
+  def missing
+    UserMailer.missing('claudio.kerekes@rootstrap.com')
```

Comment:
maybe use faker #email?

Reply:
I will add the Faker gem into the Dev mode